module_utils/network_lsr/argument_validator.py

```diff
@@ -9,6 +9,8 @@
 from ansible.module_utils.network_lsr import MyError
 from ansible.module_utils.network_lsr.utils import Util
 
+UINT32_MAX = 2 ** 32 - 1
```

Comment:
```suggestion
UINT32_MAX = 0xFFFF_FFFF
```

This might be more readable, what do you think?

Reply:
Yup, think so.